Week2/nadin.js

```diff
@@ -0,0 +1,181 @@
+'use strict'
```

Comment:
The homework is in the week2 folder but the it is week3 already!

Reply:
Thanks for the heads up! I will keep it in mind for the next time. :)